week2/intro_to_property_packages_verbose.ipynb

```diff
@@ -0,0 +1,681 @@
+    "# Property Package Introduction\n",
+    "\n",
+    "The properties of water in WaterTAP models are defined by the [property models](https://watertap.readthedocs.io/en/stable/technical_reference/property_models/index.html). \n",
+    "\n",
+    "In WaterTAP unit models, the property models representing the inlet and outlet stream are sub-models of the main unit model called _state blocks_. State blocks describe physicochemical properties of a given stream.\n",
+    "\n",
+    "The creation of state blocks is done automatically as part of the build for any unit model.\n",
+    "\n",
+    "This tutorial will cover the basics of interacting with property packages and state blocks in WaterTAP. We will do this via creation of a feed block with the `Feed` model. This model is typically the starting point for any WaterTAP flowsheet.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a799f1a2",
+   "metadata": {},
+   "source": [
+    "### Required imports from Pyomo, IDAES, and WaterTAP\n",
+    "\n",
+    "We are importing the `Feed` unit model from IDAES and will be using the [WaterTAP seawater property model](https://watertap.readthedocs.io/en/stable/technical_reference/property_models/seawater.html) as the property package for the initial demonstration. This package implements property relationships for seawater as provided in [Sharqawy et al. (2010)](https://doi.org/10.5004/dwt.2010.1079) and [Nayar et al. (2016)](https://doi.org/10.5004/dwt.2010.1079)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5820ecd5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from pyomo.environ import ConcreteModel, value, units as pyunits\n",
+    "\n",
+    "from idaes.core import FlowsheetBlock\n",
+    "from idaes.models.unit_models import Feed\n",
+    "from idaes.core.util.model_statistics import degrees_of_freedom\n",
+    "\n",
+    "from watertap.property_models.seawater_prop_pack import SeawaterParameterBlock\n",
+    "from watertap.property_models.NaCl_prop_pack import NaClParameterBlock\n",
+    "from watertap.property_models.NaCl_T_dep_prop_pack import (\n",
+    "    NaClParameterBlock as NaClParameterBlock_Tdep,\n",
+    ")\n",
+    "from watertap.core.solvers import get_solver"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0676ed18",
+   "metadata": {},
+   "source": [
+    "### Create model, flowsheet, and property model\n",
+    "\n",
+    "Like any WaterTAP model, we first create the model, flowsheet, and property model. \n",
+    "\n",
+    "<center><img src=\"graphics/prop_pack1.png\" width=\"400\" /></center>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8ca5373d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m = ConcreteModel()\n",
+    "m.fs = FlowsheetBlock(dynamic=False)\n",
+    "m.fs.properties = SeawaterParameterBlock()"
```

Comment:
separate the m.fs.properties line from the bunch to highlight the exact step where we create the instance of the property model. Want to accentuate this particular line/step, as the start of the show in this tutorial is the prop model